database/scrapers/about_us.ts

```diff
@@ -0,0 +1,143 @@
+import { DateTime } from "luxon";
+import * as fs from "node:fs/promises";
+import { Readable } from "node:stream";
+
+import { BaseScraperModule, TaskHandle } from "#commands/db_scrape";
+import { stringToLinkType } from "#enums/link_type";
+import AboutUsGeneral from "#models/about_us_general";
+import AboutUsGeneralLink from "#models/about_us_general_link";
+import FilesService from "#services/files_service";
+
+const ABOUT_US_PATH = "https://admin.topwr.solvro.pl/items/AboutUs/";
+const LINKS_JSON_PATH = "./assets/about_us_links.json";
+const ASSETS_PATH = "https://admin.topwr.solvro.pl/assets/";
+const filesMetaPath = (id: string) =>
+  `https://admin.topwr.solvro.pl/files/${id}?fields=filename_disk`;
+
+interface SourceResponse<T> {
+  data: T[];
+}
+
+interface AboutUsDraft {
+  id: number;
+  description: string;
+  cover: string | null;
+  solvroSocialLinks: number[];
+  date_updated: string;
+  user_updated: string;
+}
+
+interface SocialLinkDraft {
+  linkType: string;
+  link: string;
+}
+
+export default class AboutUsScraper extends BaseScraperModule {
+  static name = "`About Us` general information";
+  static description = "Updates about us information from Directus";
+  static taskTitle = "Update `About Us` information";
+
+  async shouldRun(): Promise<boolean> {
+    return await this.modelHasNoRows(AboutUsGeneral, AboutUsGeneralLink);
+  }
+
+  async run(task: TaskHandle) {
+    task.update("Fetching About Us data...");
+    const body = (await this.fetchJSON(ABOUT_US_PATH, "About Us data")) as {
+      data: AboutUsDraft;
+    };
+    task.update("Filling the database...");
+    const draft = body.data;
+    const formattedAboutUsData = {
+      description: draft.description,
+      coverPhotoKey: await this.semaphore.runTask(() =>
+        this.uploadCoverAndGetKey(draft).then((value) => value ?? undefined),
+      ),
+      createdAt: resolveDate(draft.date_updated), //no date created provided in the response
+      updatedAt: resolveDate(draft.date_updated),
+    };
+    await AboutUsGeneral.create(formattedAboutUsData);
+    task.update("About Us created!");
+    task.update(`Loading About Us links from ${LINKS_JSON_PATH}...`);
+    const linkData = await fs
+      .readFile(LINKS_JSON_PATH, { encoding: "utf-8" })
+      .then(JSON.parse)
+      .then((data) => {
+        if (!isValidLinkData(data)) {
+          throw new Error(`
+            Invalid JSON structure in ${LINKS_JSON_PATH},
+            expected type of Response<SocialLinkDraft>
+            `);
+        }
+        return data;
+      });
+    await AboutUsGeneralLink.createMany(
+      linkData.data.map((linkDraft) => {
+        return {
+          ...linkDraft,
+          linkType: stringToLinkType(linkDraft.linkType),
+        };
+      }),
+    );
```

Comment:
this `stringToLinkType` conversion function is purely to please TS, isn't it?
i'm sure the database wouldn't care.
(this might even make the potential database errors more cryptic, as you'll likely get a "required column missing error" instead of "this value is not a valid enum value", since your conversion functions converts invalid enum values to `undefined`)